src/pybind/mgr/dashboard/frontend/src/app/ceph/block/block.module.ts

```diff
@@ -36,6 +33,9 @@ import { RbdTrashListComponent } from './rbd-trash-list/rbd-trash-list.component
 import { RbdTrashMoveModalComponent } from './rbd-trash-move-modal/rbd-trash-move-modal.component';
 import { RbdTrashPurgeModalComponent } from './rbd-trash-purge-modal/rbd-trash-purge-modal.component';
 import { RbdTrashRestoreModalComponent } from './rbd-trash-restore-modal/rbd-trash-restore-modal.component';
+import { SharedModule } from '~/app/shared/shared.module';
+import { FeatureTogglesGuardService } from '~/app/shared/services/feature-toggles-guard.service';
+import { ActionLabels, URLVerbs } from '~/app/shared/constants/app.constants';
```

Comment:
Any clue why these are moved to the bottom? Usually absolute imports come before than relative imports, and in this case they're reversed... 🤔 I'd suggest tuning this rule to keep absolute import firsts.